perf: ⚡️ add slim image variety (#50)
MCDReforged/MCDReforged-Docker: README.md

## Image Variants

> **Compatibility warning: If the third-party Python packages you want to install require compiling from source, please resolve the issue on your own or avoid using the image with the `slim` tag.**
> 

Reviewer: ```suggestion
```

### `<version>-plus`

This image installed some useful plugins, the plugins list is below.

If you would like to add any plugins, please open an [poll](https://github.com/MCDReforged/MCDReforged-Docker/discussions/new?category=polls).

| Name               | Version |
| ------------------ | ------- |
| Quick Backup Multi | 1.9.0   |
| Minecraft Data API | 1.4.1   |
| AdvancedCalculator | 0.3.1   |
| Where Is           | 2.0.2   |

### `<version>-slim`

> This image does not contain the common linux packages, and only contains the minimal packages needed to run MCDReforged and your server.
> 
> Compare the size of the default image, this image is reduced by about `500MB`

Reviewer: 将对比省略是因为作为变体，这不应该是一个推荐的选择

```suggestion
This image is based on the python slim image and does not contain the common Linux packages, only the minimum packages needed to run MCDReforged and Minecraft server.

**Compatibility warning**: You may need to install some packages or take some risk of incompatibility.
```

> ### `<version>-slim-plus`
> 
> This image is based on `<version>-slim` and installed the same plugins as `<version>-plus`.

Reviewer: ```suggestion
The `<version>-slim-plus` image is based on `<version>-slim` and installed the same plugins as `<version>-plus`.
```

If you would like to add any plugins, please open an [poll](https://github.com/MCDReforged/MCDReforged-Docker/discussions/new?category=polls).

> | Name               | Version |
> | ------------------ | ------- |
> | Quick Backup Multi | 1.9.0   |
> | Minecraft Data API | 1.4.1   |
> | AdvancedCalculator | 0.3.1   |
> | Where Is           | 2.0.2   |

Reviewer: ```suggestion
| Name | Version |
| - | - |
| Quick Backup Multi | 1.9.0 |
| Minecraft Data API | 1.4.1 |
| AdvancedCalculator | 0.3.1 |
| Where Is | 2.0.2 |
```

Author: 这是自动格式化的锅:(
